.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node: ['15', '14', '12']
```

Comment:
I would probably use `'*'` instead of `'15'` to better approximate the `node` entry from the travis config.

This will use the latest node, though it needs to be cached first, so with some delay. Presently, it resolves to `14.15.0`.

Reply:
I looked at a way to approximate the `node` entry but couldn't find it. I tried `latest` but to no avail. Thanks Gil.